js/src/geometries/PlainGeometry.js

```diff
@@ -0,0 +1,22 @@
+//
+// This file auto-generated with generate-wrappers.js
```

Comment:
this comment is maybe wrong? if this file is auto-gen'ed, it should have the autogen extension, no?